examples/fastapi/advanced.py

```
@@ -0,0 +1,432 @@
+            "languages": translation_service.LANGUAGE_CODES,
+            "total_count": len(translation_service.LANGUAGE_CODES),
+            "premium_user": True
+        }
+    else:
+        # 비인증 사용자에게는 기본 언어만 제공
+        return {
+            "languages": basic_languages,
+            "total_count": len(basic_languages),
+            "premium_user": False,
+            "note": "API 키를 제공하면 더 많은 언어를 이용할 수 있습니다."
+        }
```

Pylint (3.3.7)

[refactor] Unnecessary "else" after "return", remove the "else" and de-indent the code inside it

(R1705)